app/src/main/res/layout/activity_logic_analyzer.xml

```diff
@@ -1,9 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/la_frame_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
-    android:padding="20dp">
```

Comment:
https://drive.google.com/file/d/0B0Nwrte8-rWVNUZtZkRhc0VWUHc/view?usp=sharing
here is the link to the apk